<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="AuthenticationException.xml" source-language="en-US" target-language="ko-KR">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac5aa6178af6b5993724548fc74499fb4c32b360c65.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">aa6178af6b5993724548fc74499fb4c32b360c65</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netcore-1.0,netcore-1.1,netcore-2.0,netcore-2.1,netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7,netstandard-2.0,xamarinandroid-7.1,xamarinios-10.8,xamarinmac-3.0</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.Security.Authentication.AuthenticationException">
          <source>The exception that is thrown when authentication fails for an authentication stream.</source>
          <target state="translated">인증 스트림에 대한 인증이 실패한 경우 throw되는 예외입니다.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Security.Authentication.AuthenticationException">
          <source>The <ph id="ph1">&lt;xref:System.Net.Security.NegotiateStream&gt;</ph> and <ph id="ph2">&lt;xref:System.Net.Security.SslStream&gt;</ph> classes throw this exception when the client or server cannot be authenticated.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Net.Security.NegotiateStream&gt;</ph> 및 <ph id="ph2">&lt;xref:System.Net.Security.SslStream&gt;</ph> 클래스는 클라이언트 또는 서버를 인증할 수 없는 경우이 예외를 throw 합니다.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Security.Authentication.AuthenticationException">
          <source>When this exception is thrown, you can retry the authentication with different credentials.</source>
          <target state="translated">이 예외가 throw 되 면 다른 자격 증명으로 인증을 다시 시도할 수 있습니다.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Security.Authentication.AuthenticationException">
          <source>If you cannot retry the authentication, a <ph id="ph1">&lt;xref:System.Security.Authentication.InvalidCredentialException&gt;</ph> is thrown instead of the <ph id="ph2">&lt;xref:System.Security.Authentication.AuthenticationException&gt;</ph>.</source>
          <target state="translated">인증을 다시 시도 수 없는 경우는 <ph id="ph1">&lt;xref:System.Security.Authentication.InvalidCredentialException&gt;</ph> 대신 throw 되는 <ph id="ph2">&lt;xref:System.Security.Authentication.AuthenticationException&gt;</ph>합니다.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Security.Authentication.AuthenticationException">
          <source><ph id="ph1">&lt;xref:System.Net.Security.NegotiateStream&gt;</ph> &amp;#124; <ph id="ph2">&lt;xref:System.Net.Security.SslStream&gt;</ph></source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Net.Security.NegotiateStream&gt;</ph> &amp;#124; <ph id="ph2">&lt;xref:System.Net.Security.SslStream&gt;</ph></target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" uid="T:System.Security.Authentication.AuthenticationException">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Security.Authentication.AuthenticationException" /&gt;</ph> class.</source>
          <target state="translated"><ph id="ph1">&lt;see cref="T:System.Security.Authentication.AuthenticationException" /&gt;</ph> 클래스의 새 인스턴스를 초기화합니다.</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" uid="M:System.Security.Authentication.AuthenticationException.#ctor">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Security.Authentication.AuthenticationException" /&gt;</ph> class with no message.</source>
          <target state="translated">메시지를 사용하지 않고 <ph id="ph1">&lt;see cref="T:System.Security.Authentication.AuthenticationException" /&gt;</ph> 클래스의 새 인스턴스를 초기화합니다.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Security.Authentication.AuthenticationException.#ctor">
          <source>This constructor does nothing.</source>
          <target state="translated">이 생성자는 아무 작업도 수행 합니다.</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" uid="M:System.Security.Authentication.AuthenticationException.#ctor(System.String)">
          <source>A <ph id="ph1">&lt;see cref="T:System.String" /&gt;</ph> that describes the authentication failure.</source>
          <target state="translated">인증 오류를 설명하는 <ph id="ph1">&lt;see cref="T:System.String" /&gt;</ph>입니다.</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" uid="M:System.Security.Authentication.AuthenticationException.#ctor(System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Security.Authentication.AuthenticationException" /&gt;</ph> class with the specified message.</source>
          <target state="translated">지정된 메시지를 사용하여 <ph id="ph1">&lt;see cref="T:System.Security.Authentication.AuthenticationException" /&gt;</ph> 클래스의 새 인스턴스를 초기화합니다.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Security.Authentication.AuthenticationException.#ctor(System.String)">
          <source>This constructor initializes the <ph id="ph1">&lt;xref:System.Exception.Message%2A&gt;</ph> property with the text in the <ph id="ph2">`message`</ph> parameter.</source>
          <target state="translated">이 생성자는 <ph id="ph1">&lt;xref:System.Exception.Message%2A&gt;</ph> 속성의 텍스트로 <ph id="ph2">`message`</ph> 매개 변수입니다.</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Security.Authentication.AuthenticationException.#ctor(System.String)">
          <source>The <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property is set to <ph id="ph2">`null`</ph>.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> 속성은 <ph id="ph2">`null`</ph>로 설정됩니다.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" uid="M:System.Security.Authentication.AuthenticationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>A <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> instance that contains the information required to deserialize the new <ph id="ph2">&lt;see cref="T:System.Security.Authentication.AuthenticationException" /&gt;</ph> instance.</source>
          <target state="translated">새 <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> 인스턴스를 deserialize하는 데 필요한 정보가 들어 있는 <ph id="ph2">&lt;see cref="T:System.Security.Authentication.AuthenticationException" /&gt;</ph> 인스턴스입니다.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" uid="M:System.Security.Authentication.AuthenticationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>A <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph> instance.</source>
          <target state="translated"><ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph> 인스턴스입니다.</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" uid="M:System.Security.Authentication.AuthenticationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Security.Authentication.AuthenticationException" /&gt;</ph> class from the specified instances of the <ph id="ph2">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> and <ph id="ph3">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph> classes.</source>
          <target state="translated"><ph id="ph1">&lt;see cref="T:System.Security.Authentication.AuthenticationException" /&gt;</ph>와 <ph id="ph2">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> 클래스의 지정된 인스턴스로부터 <ph id="ph3">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph> 클래스의 새 인스턴스를 초기화합니다.</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" uid="M:System.Security.Authentication.AuthenticationException.#ctor(System.String,System.Exception)">
          <source>A <ph id="ph1">&lt;see cref="T:System.String" /&gt;</ph> that describes the authentication failure.</source>
          <target state="translated">인증 오류를 설명하는 <ph id="ph1">&lt;see cref="T:System.String" /&gt;</ph>입니다.</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" uid="M:System.Security.Authentication.AuthenticationException.#ctor(System.String,System.Exception)">
          <source>The <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph> that is the cause of the current exception.</source>
          <target state="translated">현재 예외의 원인이 되는 <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph>입니다.</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" uid="M:System.Security.Authentication.AuthenticationException.#ctor(System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Security.Authentication.AuthenticationException" /&gt;</ph> class with the specified message and inner exception.</source>
          <target state="translated">지정된 메시지 및 내부 예외를 사용하여 <ph id="ph1">&lt;see cref="T:System.Security.Authentication.AuthenticationException" /&gt;</ph> 클래스의 새 인스턴스를 초기화합니다.</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Security.Authentication.AuthenticationException.#ctor(System.String,System.Exception)">
          <source>This constructor initializes the <ph id="ph1">&lt;xref:System.Exception.Message%2A&gt;</ph> property with the text in the <ph id="ph2">`message`</ph> parameter and initializes the <ph id="ph3">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property with the <ph id="ph4">`innerException`</ph> parameter value.</source>
          <target state="translated">이 생성자는 <ph id="ph1">&lt;xref:System.Exception.Message%2A&gt;</ph> 속성의 텍스트로 <ph id="ph2">`message`</ph> 매개 변수를 초기화 합니다는 <ph id="ph3">&lt;xref:System.Exception.InnerException%2A&gt;</ph> 속성을는 <ph id="ph4">`innerException`</ph> 매개 변수 값입니다.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>